docs/how-to/uc-image-creation.rst

```diff
@@ -0,0 +1,307 @@
+
+    snap sign -k rtu-model model.json > model.signed.yaml
+
+The ``snap sign`` command takes JSON as input and produces YAML as output!
+
+.. tip::
+
+    You need to repeat the signing every time you change the input model, because the signature is calculated based on the model.
+
+
+Build the image
+~~~~~~~~~~~~~~~
+
+First, get familiar with the tooling by referring to the guide on `building Ubuntu Core images`_.
+
+We use ``ubuntu-image`` and need to set the paths to the following as input:
+
+- Exported store credentials
+- Signed model assertion YAML file
+
+Export the store credentials to a file:
+
+.. code-block:: shell
+
+    snapcraft export-login credentials.txt
+
+Then build the image:
+
+.. code-block:: console
+
+    $ UBUNTU_STORE_AUTH_DATA_FILENAME=credentials.txt ubuntu-image snap model.signed.yaml --verbose --validation=enforce
+    [0] prepare_image
+    Fetching snapd (21759)
+    Fetching realtime-kernel (149)
+    Fetching core22 (1586)
+    Fetching pc (146)
+    [1] load_gadget_yaml
+    [2] set_artifact_names
+    [3] populate_rootfs_contents
+    [4] generate_disk_info
+    [5] calculate_rootfs_size
+    [6] populate_bootfs_contents
+    [7] populate_prepare_partitions
+    [8] make_disk
+    [9] generate_snap_manifest
+    Build successful
+
+This downloads all the snaps specified in the model assertion and builds an image file called ``pc.img``.
+
+.. hint::
+
+    To fetch the ``realtime-kernel`` snap for this image build, it should be included explicitly in your dedicated Snap Store.
+
+.. code-block:: console
+
+    $ file pc.img 
+    pc.img: DOS/MBR boot sector; partition 1 : ID=0xee, start-CHS (0x0,0,0), end-CHS (0x0,0,0), startsector 1, 6195199 sectors, extended partition table (last)
+
+✅ The image file is now ready. Refer to Ubuntu Core guide on `flashing the image to a storage medium`_.
+
+----
+
+After installing this image on your device, you can continue by tuning your system for real-time processing. 
+The :doc:`../how-to/uc-boot-parameters` guide describes the method for dynamically configuring the kernel command line parameters.
+The configuration is an iterative process that is best done together with the expected workload.
+
+Once satisfied with the configurations, continue below to learn how those configurations can be set statically during the image build.
+
+.. _custom-ubuntu-core-image:
+
+Create a custom real-time Ubuntu Core image
+-------------------------------------------
+
+This section shows how to statically set the desired Kernel command-line parameters for the Ubuntu Core system.
+To do this, we need to create a custom gadget snap, create a model assertion, and then build the OS image.
+
+.. admonition:: Project directory
+
+    Start in an empty directory.
+    We refer to this in different parts of the document as our *project directory*.
+
+Create the gadget snap
+----------------------
+
+The `gadget snap`_ documentation is a recommended read before starting.
```

Comment:
Hey @farshidtz , a general comment.

The how-to can omit explanation which is not focused on RTU itself.
https://diataxis.fr/how-to-guides/#key-principles

Also it can be a lot more "you" and not "our" since this isn't a tutorial.
https://diataxis.fr/how-to-guides/#the-language-of-how-to-guides

Reply:
Thanks for the info. I've tried to avoid explanations that are **readily available** from the upstream docs.